applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.cpp

```diff
@@ -277,8 +302,8 @@ void GenericSmallStrainHighCycleFatigueLaw<TConstLawIntegratorType>::SetValue(
         GenericSmallStrainIsotropicDamage<TConstLawIntegratorType>::SetValue(rThisVariable, rValue, rCurrentProcessInfo);
     } else if (rThisVariable == FATIGUE_REDUCTION_FACTOR) {
         mFatigueReductionFactor = rValue;
-    } else if (rThisVariable == NUMBER_OF_CYCLES) {
-        mNumberOfCycles = rValue;
+    } else if (rThisVariable == WOHLER_STRESS) {
+        mWohlerStress = rValue;
```

Comment:
```suggestion
        noalias(mWohlerStress) = rValue;
```

Reply:
mWohlerStress is a double not a vector, 'noalias()' is still required?